### PR TITLE
Set user fullname correctly, initialize identity mappings fully

### PR DESCRIPTION
**Pull request by liggitt**
1. Added the primary identity username as the user's "FullName" field, so we can display what the user expects in the UI
2. Fully initialized the uid, name, and creationtimestamp fields for the useridentitymapping objects

### CODE REVIEW

**Comment by liggitt** [comment]
@deads2k review

File: pkg/user/registry/etcd/etcd.go [hunk 1]
```diff
@@ -53,7 +52,6 @@ func (r *Etcd) GetUserIdentityMapping(name string) (mapping *api.UserIdentityMap
 func (r *Etcd) CreateOrUpdateUserIdentityMapping(mapping *api.UserIdentityMapping) (*api.UserIdentityMapping, bool, error) {
```
Comment (deads2k) [inline pkg/user/registry/etcd/etcd.go]: This method used to mutate the the passed mapping.  Do callers of this method depend upon that behavior?  

Just want to be sure we check.

**Comment by deads2k** [comment]
One question about ugly old side effects (removed, yay!), but the go parts lgtm.  I don't have experience with the html templates or asset file

File: pkg/user/registry/etcd/etcd.go (hunk 1 above)
Comment (liggitt) [inline pkg/user/registry/etcd/etcd.go]: checked, all the places that call this use the returned mapping

**Comment by liggitt** [comment]
@jwforres review the html change?

**Comment by jwforres** [comment]
HTML change LGTM

**Comment by liggitt** [comment]
[merge]

**Pull request merged**